Vagrantfile

```diff
@@ -123,10 +123,14 @@ Vagrant.configure('2') do |config|
         fi
       done
       cd "$HOME"
-      nohup /opt/chefdk/embedded/bin/ruby -run -e httpd "$HOME/app_service" -p5000 2>&1 > /dev/null &
+      netstat -nl | grep :5000 > /dev/null || nohup /opt/chefdk/embedded/bin/ruby -run -e httpd "$HOME/app_service" -p5000 2>&1 > /dev/null &
```

Comment:
True, while on the data set of all listening ports should be small enough that it shouldn't be an issue, it's a better practice.